### PR TITLE
Add a full_title helper

### PR DESCRIPTION
**Pull request by NguyenDinhDuy**

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+
+  # Returns the full title on a per-page basis.       # Documentation comment
+  def full_title(page_title = '')                     # Method def, optional arg
```
Comment (dat-hedspi) [inline app/helpers/application_helper.rb]: remove ()

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -1,13 +1,12 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
+    <title><%= full_title(yield(:title)) %></title>
```
Comment (duchv2307) [inline app/views/layouts/application.html.erb]: delete ()

Comment (dat-hedspi) [inline app/views/layouts/application.html.erb]: remove ()

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all',
                                               'data-turbolinks-track': 'reload' %>
```
Comment (duchv2307) [inline app/views/layouts/application.html.erb]: thay 1tab = 1 space

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (duchv2307) [inline app/views/layouts/application.html.erb]: change ' = "

**Pull request merged**